code/estimator_22states.cpp

```diff
@@ -1140,9 +1198,7 @@ void AttPosEKF::FuseVelposNED()
 
```

Comment:
at line 1195 above, AP_NavEKF.cpp has && instead of || .
Changing this file to match seriously degrades attitude estimation performance at 124 sec for the data in f450_glitch_log.zip
It seems to me that this must be a bug in AP_NavEKF (the change to logical OR is not mine)